doc/classes/EditorDock.xml

```diff
@@ -63,9 +63,9 @@
 		</method>
 	</methods>
 	<members>
-		<member name="available_layouts" type="int" setter="set_available_layouts" getter="get_available_layouts" enum="EditorDock.DockLayout" is_bitfield="true" default="1">
-			The available layouts for this dock, as a bitmask.
-			If you want to make all layouts available, use [code]available_layouts = DOCK_LAYOUT_VERTICAL | DOCK_LAYOUT_HORIZONTAL[/code].
+		<member name="available_layouts" type="int" setter="set_available_layouts" getter="get_available_layouts" enum="EditorDock.DockLayout" is_bitfield="true" default="5">
+			The available layouts for this dock, as a bitmask. By default, the dock allows vertical and floating layouts.
+			If you want to make all layouts available, use [code]available_layouts = DOCK_LAYOUT_VERTICAL | DOCK_LAYOUT_HORIZONTAL | DOCK_LAYOUT_FLOATING[/code].
```

Comment:
One could argue the combination of all flags should also be a constant, but eh.
Maybe? Don't have to accept it, it's a nitpick.
```suggestion
			If you want to make all layouts available, assign [code]DOCK_LAYOUT_VERTICAL | DOCK_LAYOUT_HORIZONTAL | DOCK_LAYOUT_FLOATING[/code] to this property.
```